include/picongpu/particles/functor/misc/Rng.hpp

```diff
@@ -51,7 +51,7 @@ namespace picongpu
                     using Distribution = T_Distribution;
                     using RNGFactory = pmacc::random::RNGProvider<simDim, random::Generator>;
                     using RngHandle = typename RNGFactory::Handle;
-                    using RandomGen = RngWrapper<cupla::Acc, typename RngHandle::GetRandomType<Distribution>::type>;
+
 
                     /** constructor
```

Comment:
Not, sure, but I think there should only be one blank line here.

Reply:
removed the double blank line